src/ServiceControl.Monitoring.UnitTests/API/AggregationTests.cs

```diff
@@ -43,7 +44,52 @@ public void MetricAggregationByInstanceIsScopedToLogicalEndpoint()
             Assert.AreEqual(5, model.Instances[0].Metrics["ProcessingTime"].Average);
         }
 
-        static DiagramApiController CreateConroller(ProcessingTimeStore processingTimeStore, EndpointRegistry endpointRegistry)
+        [Test]
+        public void ValidateIfConnectivityMathIsCorrect()
+        {
+            var processingTimeStore = new ProcessingTimeStore();
+            var endpointRegistry = new EndpointRegistry();
+
+            var apiController = CreateController(processingTimeStore, endpointRegistry, out var activityTracker);
```

Comment:
Another option would be to have the activity tracker be a default parameter and then the static method creates it locally if not passed. This then allows to pass it in when needed without needing to do the crazy out trick